packages/types/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@appium/types",
+  "version": "0.0.0",
```

Comment:
this will actually be published to NPM, correct? should we start at 0.0.1?

Reply:
I figured we could just update that when publishing.